netbox/templates/dcim/device_component_add.html

```diff
@@ -20,6 +20,11 @@ <h4>{{ component_type|title }}</h4>
         <div class="col-md-8 text-end">
         {% block buttons %}
         <a class="btn btn-outline-danger" href="{{ return_url }}">Cancel</a>
+        {% if perms.ipam.add_ipaddress %}
```

Comment:
Only concern, and I haven't pulled this down myself to test, does this show for all components or only interface?

Reply:
You make a good point, I’ll review and adjust if necessary.

Reply:
04cc43b should fix this